crates/glaredb_core/src/functions/scalar/builtin/binary/bitand.rs

```diff
@@ -0,0 +1,146 @@
+use std::fmt::Debug;
+use std::marker::PhantomData;
+use std::ops::BitAnd;
+
+use glaredb_error::Result;
+
+use crate::arrays::array::Array;
+use crate::arrays::array::physical_type::{
+    MutableScalarStorage,
+    PhysicalI8,
+    PhysicalI16,
+    PhysicalI32,
+    PhysicalI64,
+    PhysicalI128,
+    PhysicalU8,
+    PhysicalU16,
+    PhysicalU32,
+    PhysicalU64,
+    PhysicalU128,
+};
+use crate::arrays::batch::Batch;
+use crate::arrays::datatype::{DataType, DataTypeId};
+use crate::arrays::executor::OutBuffer;
+use crate::arrays::executor::scalar::BinaryExecutor;
+use crate::expr::Expression;
+use crate::functions::Signature;
+use crate::functions::documentation::{Category, Documentation, Example};
+use crate::functions::function_set::ScalarFunctionSet;
+use crate::functions::scalar::{BindState, RawScalarFunction, ScalarFunction};
+
+pub const FUNCTION_SET_BITAND: ScalarFunctionSet = ScalarFunctionSet {
```

Comment:
Add a comment on these functions sets say that they're not added to default list of functions to avoid conflicts with the aggregate `bit_and` and `bit_or` functions.